host/xtest/xtest_20000.c

```diff
@@ -112,14 +112,14 @@ static const struct xtest_enc_fs_case xtest_enc_fs_cases[] = {
 	XTEST_ENC_FS(1, 4094, META1, BLOCK0, VERSION0, VERSION1),
 	XTEST_ENC_FS(1, 4095, META1, BLOCK0, VERSION0, VERSION1),
 	XTEST_ENC_FS(0, 4097, META1, BLOCK0, VERSION0, VERSION1),
-	XTEST_ENC_FS(0, 4097, META1, BLOCK1, VERSION0, VERSION1),
+	XTEST_ENC_FS(0, 4097, META1, BLOCK1, VERSION1, VERSION1),
 	XTEST_ENC_FS(1, 4098, META1, BLOCK0, VERSION0, VERSION1),
-	XTEST_ENC_FS(1, 4098, META1, BLOCK1, VERSION0, VERSION1),
-	XTEST_ENC_FS(1, 1*4096, META1, BLOCK1, VERSION0, VERSION1),
-	XTEST_ENC_FS(1, 2*4096, META1, BLOCK2, VERSION0, VERSION1),
-	XTEST_ENC_FS(1, 3*4096, META1, BLOCK3, VERSION0, VERSION1),
-	XTEST_ENC_FS(1, 4*4096, META1, BLOCK3, VERSION0, VERSION1),
-	XTEST_ENC_FS(1, 4*4096, META1, BLOCK4, VERSION0, VERSION1),
+	XTEST_ENC_FS(1, 4098, META1, BLOCK1, VERSION1, VERSION1),
+	XTEST_ENC_FS(1, 1*4096, META1, BLOCK1, VERSION1, VERSION1),
+	XTEST_ENC_FS(1, 2*4096, META1, BLOCK2, VERSION1, VERSION1),
+	XTEST_ENC_FS(1, 3*4096, META1, BLOCK3, VERSION1, VERSION1),
+	XTEST_ENC_FS(1, 4*4096, META1, BLOCK3, VERSION1, VERSION1),
+	XTEST_ENC_FS(1, 4*4096, META1, BLOCK4, VERSION1, VERSION1),
```

Comment:
I don't understand these changes.
Would it be meaningful to keep the old ones **and** add these new cases ?